hack/docker-push.sh

```diff
@@ -80,4 +80,4 @@ unset GIT_HTTP_USER_AGENT
 
 echo "Building and pushing container images."
 
-VERSION="$VERSION" TAG_LATEST="$TAG_LATEST" make all-containers all-push
+VERSION="$VERSION" TAG_LATEST="$TAG_LATEST" make all-build all-containers all-push all-manifests
```

Comment:
do we need to add `all-build` here? I think the `all-containers` prerequisite chain is sufficient for getting the binaries built prior to building the containers

Reply:
should be fine as-is.